src/rocm_docs/projects.py

```diff
@@ -295,9 +295,9 @@ def _update_theme_configs(
     app: Sphinx, current_project: Optional[_Project], current_branch: str
 ) -> None:
     """Update configurations for use in theme.py"""
-    latest_version = "5.6.1"
+    latest_version = "5.7"
     latest_version_string = f"docs-{latest_version}"
-    release_candidate = "5.7"
+    release_candidate = "5.8"
```

Comment:
6.0 is the next version

Reply:
Thanks. Fixed in [de84f5d](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/381/commits/de84f5db304b27fd4415df39ad80df4f25c33c10)